os/net/ipv6/multicast/mpl.c

```diff
@@ -483,6 +493,7 @@ buffer_free(struct mpl_msg *msg)
   if(trickle_timer_is_running(&msg->tt)) {
     trickle_timer_stop(&msg->tt);
   }
+  list_remove(msg->seed->min_seq, msg);
```

Comment:
This looks suspicious. Needed for both modes?

Reply:
That was a preexisting bug within the `buffer_free` function that caused no problems with reactive forwarding always enabled, however became an issue when it was disabled. Should that have been done in a separate pull request?

Reply:
> Should that have been done in a separate pull request?

No. Just making sure it does not break anything